src/hooks/listHooks.js

```diff
@@ -64,6 +65,31 @@ export const useRenderItem = (query, navigation, options = {}) => {
       );
       break;
     }
+    case LIST_TYPES.TEXT_WITH_BIG_CARD_LIST: {
+      renderItem = ({ item, index }) => {
+        if (index === 0) {
+          return (
+            <BigCardListItem navigation={navigation} horizontal={options.horizontal} item={item} />
+          );
+        } else {
+          return (
+            <TextListItem
+              item={{
+                ...item,
+                bottomDivider: true
+              }}
+              {...{
+                navigation,
+                noSubtitle: options.noSubtitle,
+                rightImage: true,
+                textListItemWithBigCard: true
+              }}
+            />
```

Comment:
what are you thinking about this alternative from readability aspects?

```
<TextListItem
  item={{
    ...item,
    bottomDivider: true
  }}
  navigation={navigation}
  noSubtitle={options.noSubtitle}
  rightImage
  withCard
/>
```

Reply:
Since the other props are also in this way, I used it in this way for code integrity, but I will fix it to increase readability.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/74e7eb7f2b529145ccbb1f97a902aa1981ad5341